src/Adapters/Phpunit/Iteration.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\Collision\Adapters\Phpunit;
+
+/**
+ * @internal
+ */
+class Iteration
+{
+    /**
+     * The current iteration of a repeated test.
+     *
+     * @readonly
+     *
+     * @var int|null
```

Comment:
Why does it have to be `null`?

```suggestion
     * @var int
```